src/app/Unauthenticated.tsx

```diff
@@ -1,7 +1,53 @@
-import Login from 'pages/Login';
+import { Card, CardContent, Grid, Typography } from '@mui/material';
+import { Auth } from '@supabase/ui';
+import Topbar from 'components/header/Topbar';
+import { FormattedMessage } from 'react-intl';
+import { supabase } from 'services/supabase';
+import { getLoginSettings } from 'utils/env-utils';
 
 const Unauthenticated = () => {
-    return <Login />;
+    const loginSettings = getLoginSettings();
+
+    return (
+        <Grid
+            container
+            spacing={0}
```

Comment:
The default value for the `spacing` property is 0. Is there a reason to set it explicitly here?

Reply:
Nope... I thought the default was 1